docs/en/serverless/quickstarts/k8s-logs-metrics.mdx

```diff
@@ -0,0 +1,45 @@
+---
+slug: /serverless/observability/quickstarts/k8s-logs-metrics
+title: Kubernetes logs and metrics
+description: Learn how to monitor your cluster infrastructure running on Kubernetes.
+tags: [ 'serverless', 'observability', 'how-to' ]
+---
+
+<p><DocBadge template="technical preview" /></p>
+
+In this quickstart guide, you'll learn how to create the Kubernetes resources that are required to monitor your cluster infrastructure. 
+
+This new approach requires minimal configuration and provides you with an easy setup to monitor your cluster. You no longer need to download, install, or configure the Elastic Agent, everything happens automatically when you run the kubectl command.
+
+The kubectl command installs the Elastic Agent in your Kubernetes cluster, downloads all the Kubernetes resources needed to collect metrics from the cluster, and sends it to Elastic.
+
+## Before you begin
+
+This quickstart has the following requirements:
+
+- The **Admin** role or higher is required to onboard system logs and metrics. To learn more, refer to <DocLink slug="/serverless/general/assign-user-roles" />.
+- You have a running Kubernetes cluster.
+- You have installed Kubectl. 
+
+## Collect your data
+
+1. <DocLink slug="/serverless/observability/create-an-observability-project">Create a new ((observability)) project</DocLink>, or open an existing one.
+1. In your ((observability)) project, go to **Add Data**.
+1. Select **Monitor infrastructure**, and then select **Kubernetes**.
+1. Copy the command that's shown. For example:
+  ![Quickstart showing command for kubernetes](../images/quickstart-k8s-command.png)
+1. Open a terminal on the cluster you want to monitor, and run the command.
+
+  The kubectl command generates an output similar to the following one:
+
+  ![Kubernetes command output](../images/quickstart-k8s-command-output.png)
+
+1. Go back to the **Add Observability Data** page, where you will see the message **We are monitoring your cluster**. 
```

Comment:
It's important to note here that user might see a loading spinner first and they would need to way a bit for the data to come in.